### ISSUE TITLE
generated extension cannot be compiled 

### ISSUE THREAD

**Issue by akosyakov** [theia-ide/generator-theia-extension#43]
In order to reproduce
```bash
npm install -g yo generator-theia-extension
yo theia-extension my-extension
```

You will get a result as:
```
error TS18003: No inputs were found in config file '/workspace/my-extension/my-extension/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '["lib"]'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by akosyakov** [comment]
I cannot reproduce it from sources. @vince-fugnitto Was the latest master state published already?

**Comment by akosyakov** [comment]
ok, i see it was published 2 months ago: https://www.npmjs.com/package/generator-theia-extension/v/0.1.14

I will do a new release

**Comment by akosyakov** [comment]
published 0.1.15 with a fix
